zgw/search/build.gradle

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
The entire search module shouldn't be in the 'zgw' module. But should be a top level module. Since this new search module will, in the  future, be used for both searching for object in objectAPI but also the Valtimo cases